site/documentation/content/admin-guide/hono-kafka-client-configuration.md

```diff
@@ -141,6 +141,19 @@ The properties must be prefixed with `HONO_KAFKA_COMMONCLIENTCONFIG_` and `hono.
 A property with the same name defined in the configuration of one of the specific client types above will have precedence
 over the common property.
 
+## Kafka client metrics configuration
+
+Protocol adapters and the Command Router component by default report a set of metrics concerning the Kafka clients used for sending and receiving messages.
+
+The metrics support can be configured using the following environment variables or corresponding command line options:
+
+| Environment Variable<br>Command Line Option | Mandatory | Default | Description                                                             |
+| :------------------------------------------ | :-------: | :------ | :-----------------------------------------------------------------------|
+| `HONO_KAFKA_METRICS_ENABLED`<br>`--hono.kafka.metrics.enabled` | no | `true` | If set to `false`, no Kafka client metrics will be reported.  |
+| `HONO_KAFKA_METRICS_USE_DEFAULT_METRICS`<br>`--hono.kafka.metrics.useDefaultMetrics` | no | `true` | If set to `true`, a set of Kafka consumer and producer related default metrics will be reported. Additional metrics can be added via the `HONO_KAFKA_METRICS_METRICS_PREFIXES` property described below. |
```

Comment:
`HONO_KAFKA_METRICS_METRICS_PREFIXES` should be `HONO_KAFKA_METRICS_METRICSPREFIXES`, I guess.

It would also be good if you could rename `HONO_KAFKA_METRICS_USE_DEFAULT_METRICS` to `HONO_KAFKA_METRICS_USEDEFAULTMETRICS` so that it can be bound correctly by Quarkus.